build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -109,12 +109,14 @@ https://github.com/dart-lang/build/blob/master/docs/faq.md#how-can-i-resolve-ski
   // These can be consumed for hot reloads.
   var moduleDigests = <String, String>{
     for (var jsId in transitiveJsModules)
-      jsId.path.replaceFirst('lib/', 'packages/${jsId.package}/'):
-          '${await buildStep.digest(jsId)}',
+      _moduleDigestKey(jsId): '${await buildStep.digest(jsId)}',
   };
   await buildStep.writeAsString(appDigestsOutput, jsonEncode(moduleDigests));
 }
 
+String _moduleDigestKey(AssetId jsId) =>
+    '${ddcModuleName(jsId)}$jsModuleExtension';
```

Comment:
`ddcModuleName` will also strip the extension, which I assume would break things here?

Reply:
that's why I'm putting it back... did I introduce a bug with this?

Reply:
oh, didn't catch you were adding it back, sgtm